scripts/ci_install.sh

```diff
@@ -55,6 +55,12 @@ if [[ $GROUP == js-debugger ]]; then
     pip install xeus-python">=0.9.0,<0.10.0"
 fi
 
+# The UI tests use ipykernel with support for debugging, to not
+# extra launcher items and other UI elements to the reference screenshots
+if [[ $GROUP == ui-tests ]]; then
```

Comment:
It will then be possible to remove this block when the final release of `ipykernel` 6 is out